middleware/proxy_1_11.go

```diff
@@ -0,0 +1,27 @@
+// +build go1.11
+
+package middleware
+
+import (
+	"fmt"
+	"github.com/labstack/echo"
+	"net/http"
+	"net/http/httputil"
+)
+
+func proxyHTTP(t *ProxyTarget, c echo.Context, config ProxyConfig) http.Handler {
+	proxy := httputil.NewSingleHostReverseProxy(t.URL)
+	proxy.ErrorHandler = func(resp http.ResponseWriter, req *http.Request, err error) {
+		descr := t.URL.String()
+		if len(t.Name) > 0 {
+			descr = fmt.Sprintf("%s(%s)", t.Name, t.URL.String())
+		}
+		c.Logger().Warnf("remote %s unreachable, could not forward: %v", descr, err)
```

Comment:
Looks like this is an error, what do you think?

Reply:
Depends what we understand as an error, right? The issue is that the remote is not available is automatically an configuration problem, or a wrong written code of the application. I would say no, but I also can understand if you go for `error` rather than `warn`. Your call. 😃

Reply:
Well configuration is one of the case, it could also be possible where upstream server is not available. I would stick to error as we cannot recover from this.